### PR TITLE
Add files to enable ansible deployment

### PR DESCRIPTION
**Pull request by gregcorbett**
as requested by Indigo SQA. This PR also updates the documentation accordingly.

Despite `ansible.yaml` looking similar to `docker-compose.yaml` the two have slightly different syntax so cant be shared :(. However `ansible-playbook` can use the same `yaml/*.env` files after removing the empty lines.

`ansible-playbook` also doesn't seem to support relative file paths for mounting directories into the container (as `docker-compose` does), so I have had to put a placeholder in the file and a note in the documentation to change this before deploying.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,11 +83,24 @@ See [Install Docker Compose](https://docs.docker.com/compose/install/) for detai
    chown -R root docker/etc/cron.d
    ```
 
-8. Launch the containers. It is recommeded to wait 1 minute in order for each container to configure fully before launching the next
+8. Launch the containers.
+
+- With Docker Compose:
+
+   It is recommeded to wait 1 minute in order for each container to configure fully before launching the next
```
Comment (tofu-rocketry) [inline README.md]: Is there a way to check the status of a container before starting the next?

File: README.md [hunk 2]
```diff
@@ -79,11 +83,24 @@ See [Install Docker Compose](https://docs.docker.com/compose/install/) for detai
...
    docker-compose -f yaml/docker-compose.yaml up -d --force-recreate apel_mysql
+   # Wait 1 minute
    docker-compose -f yaml/docker-compose.yaml up -d --force-recreate apel_server
+   # Wait 1 minute
    docker-compose -f yaml/docker-compose.yaml up -d --force-recreate apel_rest_interface
    ```
 
+- With Ansible:
+
+   First, replace `<working directory>` in `yaml/ansible.yaml` with the absoulte file path of inflated source code, then run the below command.
```
Comment (tofu-rocketry) [inline README.md]: "absolute"

"inflated source code"?

I prefer "command below"

File: yaml/ansible.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
+---
+- name: APEL for Indigo
+  hosts: localhost
+  tasks:
+    # tasks file for ansible-dockerized-apel-db
```
Comment (tofu-rocketry) [inline yaml/ansible.yml]: What's a tasks file?

File: yaml/ansible.yml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        minutes: 1
+
+    # tasks file for ansible-dockerized-apel-rest-interface
+    - name: Launch APEL REST Interface Container
+      docker_container:
+        image: indigodatacloud/accounting:1.4.0-1
+        name: apel_rest_interface
+        links: apel_mysql
+        ports:
+          - "80:80"
```
Comment (tofu-rocketry) [inline yaml/ansible.yml]: Why the colon notation? Is this a mapping?

File: yaml/ansible.yml (hunk 3 above)
Comment (gregcorbett) [reply yaml/ansible.yml]: to be honest, I'm not sure. I just followed the example [here](https://www.ansible.com/blog/six-ways-ansible-makes-docker-compose-better)

File: yaml/ansible.yml (hunk 4 above)
Comment (gregcorbett) [reply yaml/ansible.yml]: yes, it maps port 80 on the host to port 80 in the container allowing the web request to be handled by the container

File: README.md (hunk 2 above)
Comment (gregcorbett) [reply README.md]: changes made.

`README.md` says to download and unzip the source code as it contains schemas and yaml files needed for deploying via docker. And 'inflating' is the the verb unzip uses.

File: yaml/ansible.yml (hunk 3 above)
Comment (tofu-rocketry) [reply yaml/ansible.yml]: Just call them "tasks" then as that seems to be the correct Ansible terminology.

File: README.md (hunk 2 above)
Comment (tofu-rocketry) [reply README.md]: I'd rather keep it more generic then: "decompressed"

File: README.md (hunk 1 above)
Comment (gregcorbett) [reply README.md]: Yes and no, mostly no. 

In theory there are ways to do it (like the `ansible` [wait_for](http://docs.ansible.com/ansible/latest/wait_for_module.html) module or `docker-compose`'s [healthcheck](https://docs.docker.com/compose/compose-file/#healthcheck)), but the main problem is the standard image database container reports it's ready when the port is opened which isn't the same as ready to accept connections, which tends to scupper those options

I can get rid of the wait between the Server and the REST interface as they are independent.

File: yaml/ansible.yml (hunk 3 above)
Comment (gregcorbett) [reply yaml/ansible.yml]: changed

File: README.md (hunk 2 above)
Comment (gregcorbett) [reply README.md]: chnaged

**Comment by gregcorbett** [comment]
changes made

File: yaml/ansible.yml [hunk 5]
```diff
@@ -2,7 +2,7 @@
 - name: APEL for Indigo
   hosts: localhost
   tasks:
-    # tasks file for ansible-dockerized-apel-db
+    # tasks for ansible-dockerized-apel-db
```
Comment (tofu-rocketry) [inline yaml/ansible.yml]: Ah, these should be "task" singular then.

Comment (gregcorbett) [reply yaml/ansible.yml]: ah yes, because each one (database, server, REST) is an individual task

Comment (gregcorbett) [reply yaml/ansible.yml]: changed

**Comment by gregcorbett** [comment]
I'll hold off the merge for now, to try and get a thumbs up from Indigo SQA next week

**Comment by gregcorbett** [comment]
I have not heard anything so I am assuming everything is awesome.

**Pull request merged**